### ISSUE TITLE
[Feature request] Customizable invoice templates

### ISSUE THREAD

**Issue by djzwerg** [backdrop-contrib/ubercart#398]
The way [described here](https://www.drupal.org/docs/7/modules/ubercart/howtos/how-to-customize-the-order-invoice-customer-e-mail) doesn't work with the BD version of Ubercart.

It would be nice to have a documented way to use custom invoice templates or to do this via the UI.

**Comment by bugfolder** [comment]
> The way [described here](https://www.drupal.org/docs/7/modules/ubercart/howtos/how-to-customize-the-order-invoice-customer-e-mail) doesn't work with the BD version of Ubercart.

Which of the three methods were you trying? Backdrop's handling of templates in themes is basically the same as Drupal's, so those Drupal-specific instructions should still work.(*)

For example, if you want to customize the invoice template, move a copy of `uc-order--customer.tpl.php` into the `templates` directory of your theme(s), edit that file, then clear all caches. Re-load the invoice page, and you'll see the updated invoice. (This is "The Simplest Way" on that docs page.)

 (*) Note that there is a new Backdrop-specific documentation site at www.ubercart.dev. It doesn't yet have this how-to page, though.

**Comment by djzwerg** [comment]
@bugfolder yes that way works - making a copy of `uc-order--customer.tpl.php`. But UC doesn't use the `uc-order--customer-bank_transfer.tpl.php` file (neither from module folder nor from theme folder).
So I think this is really an issue that the `uc-order--customer-bank_transfer.tpl.php` file won't be used by UC (issue by uc_bank_transfer module). The 'hard way' from the link above won't work, too (issue by UC module).

**Comment by argiepiano** [comment]
@djzwerg, have you selected the bank_transfer invoice template in `admin/store/settings/orders`?

<img width="489" alt="Screen Shot 2022-12-20 at 10 35 03 AM" src="https://user-images.githubusercontent.com/9938978/208730661-3100347e-f1d1-4b8c-9ac3-a8d9f1221bed.png">

**Comment by argiepiano** [comment]
There is problem with selecting the bank transfer invoice template in that path I pasted above (Order settings).  I've tried everything I can think of - placing the template in the theme's directory, and changing the name to dashes instead of underscores, without success.

I'm not sure if this is a bug with Ubercart or with UC Bank Transfer. I'll investigate later today.

**Comment by djzwerg** [comment]
@argiepiano yes that was my way to try this out.

**Comment by argiepiano** [comment]
@djzwerg I found the problem and have fixed it. I just did a new release of UC Bank Transfer. 

Important: for the template `uc-order--customer-bank_transfer.tpl.php` to work you MUST place it in your front-end theme's `template` folder. Also you must select `customer_bank_transfer` in the order settings at `admin/store/settings/orders/settings`

I've tested it and it works. If you find any issues please post a message in the UC Bank Transfer issue queue.

**Comment by djzwerg** [comment]
Thanks @argiepiano ! I'll check this after xmas. Happy holidays!

**Comment by djzwerg** [comment]
@argiepiano thanks again! Now it works for me to overwrite the bank_transfer template within my theme, too. But '[the hard way](https://www.drupal.org/docs/7/modules/ubercart/howtos/how-to-customize-the-order-invoice-customer-e-mail#s-the-hard-way)' seems to don't work in BD.
